### PR TITLE
Add support to provide envoirnment variable in action

### PR DESCRIPTION
**Pull request by Divyansh42**
Fixes: #16 
Signed-off-by: divyansh42 <diagrawa@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,8 +77,45 @@ Once an image has been built, [push-to-registry](https://github.com/redhat-actio
...
+  <tr>
+    <td>env_vars</td>
+    <td>No</td>
+    <td>-</td>
+    <td>List of Environment variable to be passed to the builder (eg. <code>key=value</code>, <code>mysecret=${{ secrets.value }}</code> ).</td>
+  </tr>
+
 </table>
 
+### Multi-line envoirnment variables
```
Comment (tetchel) [inline README.md:89]: I don't think we have to document here how to write environment variables. It follows the same rules as the regular `env` section, right?

File: README.md [hunk 2]
```diff
@@ -77,8 +77,45 @@ Once an image has been built, [push-to-registry](https://github.com/redhat-actio
     <td><a href="https://github.com/openshift/source-to-image/blob/master/docs/cli.md#log-levels">Log level</a> when running S2I. Can be 0 (least verbose) to 5 (most verbose).</td>
   </tr>
 
+  <tr>
+    <td>env_vars</td>
+    <td>No</td>
+    <td>-</td>
+    <td>List of Environment variable to be passed to the builder (eg. <code>key=value</code>, <code>mysecret=${{ secrets.value }}</code> ).</td>
```
Comment (tetchel) [inline README.md:84]: how about `${{ secrets.MY_SECRET }}`

Comment (tetchel) [reply README.md:84]: `List of environment variable key-value pairs to pass to the s2i builder context.`

File: README.md (hunk 1 above)
Comment (Divyansh42) [reply README.md:89]: Yes, it does.

**Pull request merged**